test.sh

```diff
@@ -21,19 +21,6 @@ export fn="${CUR_DIR}/fn"
 go test -v $(go list ./... |  grep -pv "^github.com/fnproject/cli/test$")
 
 
-# Our test directory
-OS=$(uname -s)
-if [ $OS = "Darwin" ]; then
-	WORK_DIR=$(mktemp -d /tmp/temp.XXXXXX)
-else
-	WORK_DIR=$(mktemp -d)
-fi
-cd $WORK_DIR
-
-# HACK: if we don't pre touch these, fn server will create these as 'root'
-mkdir data
-touch data/fn.db data/fn.mq
-
```

Comment:
did you happen to test on osx here? i'm not sure if they work on master on osx anymore with uds stuff even, just asking to keep track though.

Reply:
i see you commit msg https://github.com/fnproject/cli/pull/532/commits/8bc2e2f3f56c8da34f6ab051b76eb5485315f283 - thanks